src/core/src/sys/compute.rs

```diff
@@ -236,8 +236,8 @@ impl Compute for PressureAtTemperature {
         let virial_tensor = system.virial();
         let virial = virial_tensor.trace();
         let volume = system.volume();
-        let natoms = system.size() as f64;
-        return natoms * K_BOLTZMANN * self.temperature / volume + virial / (3.0 * volume);
+        let nmols = system.molecules().len() as f64;
```

Comment:
I'll try to write this down tomorrow, and double check in the other codes I know =)

Does this mean that computing the pressure by using the number of atoms will yield the wrong result?